02week/blog/css/style.css

```diff
@@ -0,0 +1,110 @@
+body {
```

Comment:
I didnt see any css definition that would help with the hover animation change

Reply:
What hover animation change?

Reply:
The first letter of the text is supposed to change on hover